modules/flowable-engine/src/main/java/org/flowable/engine/repository/MergeMode.java

```diff
@@ -0,0 +1,24 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.flowable.engine.repository;
+
+/**
+ * @author Valentin Zickner
+ */
+public class MergeMode {
```

Comment:
Maybe this should be an `enum`? What do you think?

Reply:
I changed it to the `enum`. Initially the idea was that people are able to extend the list and that was the reason for using a `String`. At the end I picked the approach where they can simply provide their own `DeploymentMergeStrategy` as a parameter.